eval/src/vespa/eval/tensor/sparse/sparse_tensor_index.h

```diff
@@ -40,6 +41,7 @@ class SparseTensorIndex : public vespalib::eval::Value::Index
     IndexMap _map;
     size_t _num_mapped_dims;
     static size_t needed_memory_for(const SparseTensorIndex &other);
+    SparseTensorIndex(size_t stash_size, const SparseTensorIndex &other);
```

Comment:
consider deleting the copy constructor to make sure it does not exist and force everyone to use copy/shrunk_copy.